_config.yml

```diff
@@ -26,3 +26,6 @@ defaults:
       type: "pages"
     values:
       layout: "skeleton"
+
+whitelist: [jekyll-redirect-from]
+gems: [jekyll-redirect-from]
```

Comment:
Why are these lines necessary in the `_config.yml`, but not in the `_config-asf.yml`?